webpack/assets/javascripts/react_app/components/notifications/NotificationDrawer.js

```diff
@@ -24,6 +24,7 @@ class NotificationDrawer extends Component {
 
   onChange(actionType) {
     switch (actionType) {
+      case ACTIONS.MARK_AS_READ:
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({ notifications: NotificationsStore.getNotifications() });
```

Comment:
i'm currently working on a PR that refactors everything, so no more switch statements in UI components :)